Core/Core/Discussions/DiscussionDetailsViewController.swift

```diff
@@ -237,139 +240,104 @@ extension DiscussionDetailsViewController {
         webView.loadHTMLString(webView.html(for: """
             \(Self.topicHTML(topic))
             \(topicReplyButton)
-            <div style="border-top:0.3px solid var(--color-borderMedium); margin:16px 0;"></div>
-            <h2 style="font-size:20px; font-weight:600; margin:0; padding:0;">
+            \(entries.isEmpty ? "" : """
+            <h2 class="\(Styles.heading)">
                 \(t(NSLocalizedString("Replies", bundle: .core, comment: "")))
             </h2>
+            """)
             \(entries.map { entryHTML($0, depth: 0) } .joined(separator: "\n"))
             """
         ), baseURL: topic.htmlUrl)
     }
 
     public static func topicHTML(_ topic: DiscussionTopic) -> String {
         return """
-        \(entryHeader(author: topic.author, date: topic.postedAt, isTopic: true))
+        <style>\(Self.css)</style>
```

Comment:
Having a stylesheet allows us to not repeat inline styles of entries, so we can generate large discussions faster & with less memory. Plus we can use :pseudo elements and animations in a stylesheet.